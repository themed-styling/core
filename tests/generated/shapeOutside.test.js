import 'jest-styled-components'

import { shapeOutside } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(shapeOutside, 'shapeOutside')
