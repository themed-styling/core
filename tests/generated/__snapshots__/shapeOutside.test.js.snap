// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shapeOutside renders 0 1`] = `
Array [
  .c0 {
  shape-outside: 0;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: 0;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: 0;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders function 1`] = `
Array [
  .c0 {
  shape-outside: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: has;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: has;
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders invalid theme path 1`] = `
Array [
  .c0 {
  shape-outside: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders negative number 1`] = `
Array [
  .c0 {
  shape-outside: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  shape-outside: July;
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    shape-outside: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: July;
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    shape-outside: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    shape-outside: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    shape-outside: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    shape-outside: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    shape-outside: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    shape-outside: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    shape-outside: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    shape-outside: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    shape-outside: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    shape-outside: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  shape-outside: July;
}

@media May {
  .c0 {
    shape-outside: August;
  }
}

@media April {
  .c0 {
    shape-outside: September;
  }
}

@media January {
  .c0 {
    shape-outside: November;
  }
}

@media April {
  .c0 {
    shape-outside: August;
  }
}

@media June {
  .c0 {
    shape-outside: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: July;
}

@media May {
  .c0 {
    shape-outside: August;
  }
}

@media April {
  .c0 {
    shape-outside: September;
  }
}

@media January {
  .c0 {
    shape-outside: November;
  }
}

@media April {
  .c0 {
    shape-outside: August;
  }
}

@media June {
  .c0 {
    shape-outside: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    shape-outside: November;
  }
}

@media April {
  .c0 {
    shape-outside: January;
  }
}

@media January {
  .c0 {
    shape-outside: December;
  }
}

@media April {
  .c0 {
    shape-outside: April;
  }
}

@media June {
  .c0 {
    shape-outside: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    shape-outside: November;
  }
}

@media April {
  .c0 {
    shape-outside: January;
  }
}

@media January {
  .c0 {
    shape-outside: December;
  }
}

@media April {
  .c0 {
    shape-outside: April;
  }
}

@media June {
  .c0 {
    shape-outside: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    shape-outside: July;
  }
}

@media April {
  .c0 {
    shape-outside: August;
  }
}

@media January {
  .c0 {
    shape-outside: September;
  }
}

@media April {
  .c0 {
    shape-outside: November;
  }
}

@media June {
  .c0 {
    shape-outside: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    shape-outside: July;
  }
}

@media April {
  .c0 {
    shape-outside: August;
  }
}

@media January {
  .c0 {
    shape-outside: September;
  }
}

@media April {
  .c0 {
    shape-outside: November;
  }
}

@media June {
  .c0 {
    shape-outside: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    shape-outside: January;
  }
}

@media April {
  .c0 {
    shape-outside: December;
  }
}

@media January {
  .c0 {
    shape-outside: April;
  }
}

@media April {
  .c0 {
    shape-outside: May;
  }
}

@media June {
  .c0 {
    shape-outside: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    shape-outside: January;
  }
}

@media April {
  .c0 {
    shape-outside: December;
  }
}

@media January {
  .c0 {
    shape-outside: April;
  }
}

@media April {
  .c0 {
    shape-outside: May;
  }
}

@media June {
  .c0 {
    shape-outside: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders postitive number 1`] = `
Array [
  .c0 {
  shape-outside: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders string 1`] = `
Array [
  .c0 {
  shape-outside: July;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: July;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: June;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: June;
}

<div
    className="c0"
  />,
]
`;

exports[`shapeOutside renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`shapeOutside renders valid theme path 1`] = `
Array [
  .c0 {
  shape-outside: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  shape-outside: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
